docs/nx-cloud/reference/assignment-rules.md

```diff
@@ -1,39 +1,228 @@
-# Assignment Rules (beta)
+# Assignment Rules
 
-Assignment rules allow you to control which tasks can run on which agents. Save on agent costs by provisioning different sizes of agents to suite the individual needs of your tasks. You can ensure resource intensive targets like `e2e-ci` and `build` have what they need by using larger agents. Lighter tasks like `lint` and `test` can run on smaller agents.
+Assignment rules allow you to control which tasks can run on which agents. Save on agent costs by provisioning different sizes of agents to suite the individual needs of your tasks. Ensure resource intensive targets like `e2e-ci` and `build` have what they need by using larger agents and with a specified parallelism. Lighter tasks like `lint` and `test` can run on smaller agents.
 
 Assignment rules are defined in `yaml` files within your workspace's `.nx/workflows` directory. You can use assignment rules with [Manual Distributed Task Execution (DTE)](/ci/recipes/dte) or with [dynamic Nx agents](/ci/features/dynamic-agents). Note that additional configuration is required when using Manual DTE.
 
 ## How to Define an Assignment Rule
 
-Each assignment rule has one of the following properties that it matches against tasks: `project`, `target`, and/or `configuration`. It also has a list of possible [agent types](/ci/reference/launch-templates) that tasks with the matching properties can run on. Rules are defined in yaml like the following:
+Each assignment rule has one of the following properties that it matches against tasks: `projects`, `targets`, and/or `configurations`. You can provide a list of globs to match against the tasks in your workspace. It also has a list of possible [agent types](/ci/reference/launch-templates) that tasks with the matching properties can run on. Rules are defined in yaml like the following:
+
+{% tabs %}
+{% tab label="Assignment rules with Nx Agents" %}
+
+```yaml {% fileName=".nx/workflows/distribution-config.yaml" %}
+distribute-on:
+  default: 5 linux-medium-js, 5 linux-large-js
+
+assignment-rules:
+  - projects:
+      - app1
+    targets:
+      - e2e-ci*
+    configurations:
+      - production
+    run-on:
```

Comment:
Is it `run-on` or `runs-on`?

Reply:
run-on. We are deprecating runs-on since that key does not support the parallelism object